src/main/java/net/coderbot/iris/Iris.java

```diff
@@ -100,6 +105,21 @@ public static ShaderPipeline getPipeline() {
 		return pipeline;
 	}
 
+	public static void reload() throws IOException {
+		//currently this first line can be used to reload to a diff shaderpack, but it should be removed
+		//when there is a gui or a better system for changing packs at runtime
+		//or could be kept for the gui to use
+		irisConfig.initialize();
+		Path shaderPackRoot = shaderpacksDirectory.resolve(irisConfig.getShaderPackName());
```

Comment:
What if the internal shader pack is loaded? Should probably record which shaderpack is being used somewhere during startup instead of assuming that it's an external one

I know that reloading internal shaders doesn't make a ton of sense but it shouldn't just fail hard